.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    root: true,
+    parserOptions: {
+        ecmaVersion: 6,
+        sourceType: 'module'
+    },
+    extends: 'eslint:recommended',
```

Comment:
Officially, all COS projects follow the AirBNB style guide (which is what we used for JSCS previously)
http://cosdev.readthedocs.io/en/latest/style_guides/javascript.html
https://github.com/abought/demo-ember-osf/pull/16/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L69

Presets for this exist: would it be a more appropriate choice?